docs/DeveloperGuide.md

```diff
@@ -29,6 +29,30 @@ a Command Line Interface (CLI).
 
 
 ### 1.5 Overview of Application
+In order to better understand how DonnaFin helps financial advisors, it is useful to understand and model the status
+quo without DonnaFin.
+
+Below is an Object-Oriented Domain Model (OODM) modelling the existing state of affairs between a Financial Advisor and
+their Client/s.
+
+![OverviewOodmDiagram](images/OverviewOodmDiagram.png)
```

Comment:
```suggestion
![Overview OODM Diagram](images/OverviewOodmDiagram.png)
```
the alt text is normal text usually, no need to PascalCase it.

(alt text is the portion between the `[]`)

Reply:
OODM in caps!